ee/docker-ee/ubuntu.md

```diff
@@ -93,6 +91,8 @@ and update Docker Engine - Enterprise from the repository.
 
```

Comment:
Edit the text following the **Install using the repository** subheading to read: 
 
Naturally, to install Docker Engine - Enterprise on a new host machine using the Docker repository you must first set the repository up on the machine.